packages/server/api/src/app/ai/mcp/docs.ts

```diff
@@ -0,0 +1,23 @@
+import { AppSystemProp, logger, system } from '@openops/server-shared';
+import { experimental_createMCPClient as createMCPClient } from 'ai';
+import { Experimental_StdioMCPTransport as StdioMCPTransport } from 'ai/mcp-stdio';
+
+const DOCS_MCP_SERVER_PATH =
+  system.get<string>(AppSystemProp.DOCS_MCP_SERVER_PATH) ||
+  '/root/.mcp/docs.openops.com';
```

Comment:
There's a proper way to define default values for the SystemProps.

Reply:
done